heft-plugins/heft-webpack5-plugin/src/WebpackConfigurationLoader.ts

```diff
@@ -92,3 +94,18 @@ export class WebpackConfigurationLoader {
     }
   }
 }
+
+function findWebpackConfig(buildFolder: string, configurationRegExp: RegExp): string {
+  try {
+    const configs = FileSystem.readFolderItemNames(buildFolder).filter((config) =>
```

Comment:
```suggestion
async function findWebpackConfigAsync(buildFolder: string, configurationRegExp: RegExp): Promise<string> {
  try {
    const folderItems: FolderItem = await FileSystem.readFolderItemsAsync(buildFolder);
    const configs: string[] = [];
    for (folderItem of folderItems) {
      if (folderItem.isFile() && folderItem.name.match(configurationRegExp)) {
        configs.push(folderItem.name);
      }
    }

```

Reply:
@iclanton I've incorporated this as well as an adjustment to return an object w/ dev/prod keys respectively. A bit different than the initial approach, but serves fallbacks in the case either is undefined. Let me know your thoughts when you can.